packages/mediacenter/kodi/scripts/kodi-config

```diff
@@ -42,13 +42,6 @@ if [ ! -f $HOME/.kodi/userdata/guisettings.xml ] ; then
   fi
 fi
 
-# cleanup *.fi cache files from .kodi/temp as old version
-# of these files can cause random problems/crashes, particularly
-# after an upgrade
-if [ -d $HOME/.kodi/temp ]; then
-  rm -f $HOME/.kodi/temp/*.fi
-fi
-
```

Comment:
Yep, cache files are now stored in `.kodi/temp/archive_cache` (https://github.com/xbmc/xbmc/pull/9225).

Although there's a flaw with 9225 (see comments), which https://github.com/xbmc/xbmc/pull/10206 fixes. We could add 10206 to this PR if you like, either way the above code is no longer required.